src/BlockTypes/MiniCartContents.php

```diff
@@ -74,15 +74,6 @@ protected function enqueue_assets( array $attributes ) {
 		$bg_color   = StyleAttributesUtils::get_background_color_class_and_style( $attributes );
 
 		$styles = array(
-			array(
-				'selector'   => '.wc-block-mini-cart__drawer .components-modal__header',
-				'properties' => array(
-					array(
-						'property' => 'color',
-						'value'    => $text_color ? $text_color['value'] : false,
-					),
-				),
-			),
```

Comment:
I think these styles weren't used anywhere, as we weren't setting a modal header.